tests/testthat/test-bind.R

```diff
@@ -257,6 +257,16 @@ test_that("can assign row names in vec_rbind()", {
   expect_identical(out, exp)
 })
 
+test_that("monitoring: name repair while rbinding doesn't modify in place", {
```

Comment:
What is this monitoring prefix?

Reply:
I've used it in a few places. I don't have a better way to describe it other than "this tests very internal details that aren't really specific to functionality" https://github.com/r-lib/vctrs/search?q=monitoring&unscoped_q=monitoring